Gemfile

```diff
@@ -1,10 +1,13 @@
 source 'https://rubygems.org'
 
+gem 'jbuilder'
 group :test do
-  gem 'simplecov-rcov'
+  gem 'simplecov-rcov', require: false
   #
   # TODO: change test not to use shoulda.
   #
-  gem 'shoulda'
+  gem 'shoulda', require: false
   gem 'rubocop', require: false
 end
+
+gem "codeclimate-test-reporter", group: :test, require: nil
```

Comment:
__[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)